### PR TITLE
Gateway of the Great King/Ramstis Ridge/Overlong Bridge Valley/Akmens Ridge

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -248,7 +248,7 @@ QUEST_LV_0100_20150317_000247	$Klaipeda Guard Captain
 QUEST_LV_0100_20150317_000248	$Go ahead to Klaipeda.{nl}Commander Uska is waiting for you in Klaipeda Central Plaza.{nl}I heard the Bishop also had a dream about the Goddess... it appears to concern that day.
 QUEST_LV_0100_20150317_000249	$Klaipeda Guard
 QUEST_LV_0100_20150317_000250	$Thank you for making your way here.{nl}If there is still this much left even after Sir Titas handled it himself, can you imagine how it was before?
-QUEST_LV_0100_20150317_000251	$Mercenary Tobi
+QUEST_LV_0100_20150317_000251	$Mercenary Tudou
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Wait, why was this changed to Tudou? It's Toby or Tobi.

Comment (terrapin47) [inline QUEST_LV_0100.tsv]: Oops, I forgot to fix this

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:
